webapp/src/Controller/API/UserController.php

```diff
@@ -307,12 +307,17 @@ public function addAction(
         AddUser $addUser,
         Request $request
     ): Response {
+        if ($this->eventLogService->externalIdFieldForEntity(User::class) && !$addUser->id) {
```

Comment:
Although User is not in CLICS, shouldn't we make this consistent?

Reply:
Yeah I was wondering the same.

Reply:
Updated to do this.